client/blocks/post-item/index.jsx

```diff
@@ -138,9 +140,22 @@ class PostItem extends React.Component {
 							{ isAuthorVisible && <PostTypePostAuthor globalId={ globalId } /> }
 						</div>
 						<h1 className="post-item__title">
-							<a href={ editUrl } className="post-item__title-link">
-								{ title || translate( 'Untitled' ) }
-							</a>
+							{ ! externalPostLink && (
+								<a href={ isPlaceholder ? null : postUrl } className="post-item__title-link">
+									{ title || translate( 'Untitled' ) }
+								</a>
+							) }
+							{ ! isPlaceholder &&
+							externalPostLink && (
+								<ExternalLink
+									icon={ true }
+									href={ postUrl }
+									target="_blank"
+									className="post-item__title-link"
+								>
+									{ title || translate( 'Untitled' ) }
+								</ExternalLink>
+							) }
```

Comment:
Not a big deal but I think this could be better written.  Perhaps by pulling `isPlaceholder` and `isPostLinkExternal` into `connect`ed props, then we could write:

```js
isPostLinkExternal ? (
    <ExternalLink />
) : (
    <a />
)
```

This would also allow us to get rid of the `<a href={ isPlaceholder ? null : postUrl }>` awkwardness.

Reply:
@nylen I like that suggestion.